corehq/apps/translations/integrations/transifex/transifex.py

```diff
@@ -151,7 +151,7 @@ def get_translations(self):
                                                                     self.lock_translations)
         return po_entries
 
-    def resources_pending_translations(self, break_if_true=False, all_langs=False):
+    def resources_pending_translations(self, break_if_true=True, all_langs=False):
```

Comment:
Are there any places that don't want `break_if_true=True`?  If not, I'd drop the parameter entirely.

Reply:
i had it around for debugging but i dont see a use for it anymore. removing it. 
thank you.